packages/contracts/src/utils/RelayRegistrar.sol

```diff
@@ -32,15 +32,23 @@ contract RelayRegistrar is IRelayRegistrar, ERC165 {
 
     uint256 private immutable creationBlock;
 
-    constructor() {
+    uint256 private relayRegistrationMaxAge;
```

Comment:
there is no setter. this means it is not configurable, and set hard in stone.
Since a RelayHub can't change a Registrar, it means we need to re-deploy a relayHub in order to change this value.